docs/variants/msi_z690/initial-deployment.md

```diff
@@ -2,22 +2,15 @@
 
 Initial deployment of Dasharo firmware on MSI PRO Z690-A can be done:
 
-* using DTS v1.1.0,
+* using DTS,
 * manually.
 
 ## Initial deployment using DTS
 
-Initial deployment for MSI PRO Z690-A (WIFI) (DDR4) is supported in DTS since
-version v1.1.0. Please check [Dasharo zero-touch initial deployment
-section](../../dasharo-tools-suite/documentation.md#dasharo-zero-touch-initial-deployment).
-
-<span style="color:red">
-Please do not use DTS to deploy Dasharo on MSI PRO Z690-A (WIFI) DDR5;
-otherwise, you will brick your hardware. There are at least two bugs that have
-to be fixed before it is possible:
-
-* [Power off problem](https://github.com/Dasharo/dasharo-issues/issues/316)
-* [Unsupported mainboard flashing problem](https://github.com/Dasharo/dasharo-issues/issues/317).</span>
+Please check [Dasharo zero-touch initial deployment
```

Comment:
@pietrushnic This is how I change the documentation about initial deployment. Users should always use latest DTS version. Looks ok?

Reply:
@TomaszAIR, maybe the logic of the section should be reverted. First, download the recent release and then use the initial deployment feature.

Other concerns:
- Why do we not update this section for all DTS-supported hardware platforms documented in Dasharo Universe?
- We must consider support for Dasharo Community Releases (all public releases so far) and Dasharo Supporters Releases - but that may be a feature request.

Reply:
@pietrushnic logic fixed and also populated in sections for other DTS-supported hardware https://github.com/Dasharo/docs/pull/541/commits/42b50e1c86964424a9c017811f31cffd6459f3ba - please resolve this conversation if you are ok with that.

as for Dasharo Community Releases and Dasharo Supporters Releases - please let's talk about that in other place se we can close and merge content of this PR